pertpy/tools/_distances/_distances.py

```diff
@@ -856,3 +914,97 @@ def onesided_distances(
 
     def from_precomputed(self, P: np.ndarray, idx: np.ndarray, **kwargs) -> float:
         raise NotImplementedError("ClassifierClassProjection cannot be called on a pairwise distance matrix.")
+
+
+class MeanVarDistnDistance(AbstractDistance):
+    """
+    Distance betweenv mean-var distributions of gene expression.
+
+    """
+
+    def __init__(self) -> None:
+        super().__init__()
+        self.accepts_precomputed = False
+
+    def __call__(self, X: np.ndarray, Y: np.ndarray, **kwargs) -> float:
+        """
+        Difference of mean-var distibutions in 2 matrices
+        Args:
+            X: Normalized and log transcformed cells*genes count matrix.
```

Comment:
```suggestion
            X: Normalized and log transformed cells x genes count matrix.
```

Reply:
I dont have an immediate example but think we use the small x more often than the * :)

Reply:
But once the typo is corrected, this is a very good, clear description of what X and Y are :)